sdk/eventgrid/azure-eventgrid/tests/eventgrid_preparer.py

```diff
@@ -8,12 +8,9 @@
     JsonField,
     JsonFieldWithDefault,
 )
-from azure_devtools.scenario_tests.exceptions import AzureTestError
-
-from devtools_testutils.resource_testcase import RESOURCE_GROUP_PARAM
 
 EVENTGRID_TOPIC_PARAM = "eventgrid_topic"
-EVENTGRID_TOPIC_LOCATION = "westus"
+EVENTGRID_TOPIC_LOCATION = "eastus"
```

Comment:
this prob doesn't matter much since this is just used for the fake resource url, but if you wanted to set this location to wherever the resource group is created for consistency, you can add the location to test-resources.json:
```
 "RESOURCE_GROUP_LOCATION": {
        "type": "string",
        "value": "[parameters('location')]"
      }
```
then update this to:
```suggestion
EVENTGRID_TOPIC_LOCATION = os.environ.get("RESOURCE_GROUP_LOCATION", "eastus")
```
This way you don't have to update this file if the region changes in the future.